### PR TITLE
Checking unique font family values

### PR DESCRIPTION
**Pull request by jamesbrumpton**
_This PR adds a test to pull the unique font families when the page loads. Once pulled, It will then warn us if there are too many font family files being loaded onto  the page._

* [x] Fixes https://github.com/bbc/simorgh/issues/198
* [ ] Tests added for new features

* [ ] Test engineer approval

### CODE REVIEW

File: cypress/integration/articleSpec.js [hunk 1]
```diff
@@ -89,4 +89,22 @@ describe('News Article', () => {
       .should('have.attr', 'src')
       .and('match', /(\/static\/js\/bundle\.\w+\.js)/g);
   });
+
+  it('should check the font file loads', () => {
+    const fontFamiliesArray = [];
+    cy.get('*')
```
Comment (jtart) [inline cypress/integration/articleSpec.js]: This is very inclusive and includes Cypress elements. This is why `Times` is being returned as a font. We should just get a parent element that is specific to our content.

File: cypress/integration/articleSpec.js [hunk 2]
```diff
@@ -89,4 +89,22 @@ describe('News Article', () => {
       .should('have.attr', 'src')
       .and('match', /(\/static\/js\/bundle\.\w+\.js)/g);
   });
+
+  it('should check the font file loads', () => {
```
Comment (jtart) [inline cypress/integration/articleSpec.js]: We should change this to be more specific to what is being asserted in the test, e.g. `should load less than 3 fonts`

File: cypress/integration/articleSpec.js [hunk 3]
```diff
@@ -89,4 +89,22 @@ describe('News Article', () => {
...
+    const fontFamiliesArray = [];
+    cy.get('*')
+      .each(element => {
+        const fontFamily = Cypress.$(element).css('font-family');
```
Comment (jtart) [inline cypress/integration/articleSpec.js]: Do we have to do `Cypress.$(element)`, can we just do `element.css('font-family')`?

**Review by jtart — CHANGES_REQUESTED** [review]

**Review by jtart — APPROVED** [review]

**Review by radiocontrolled — APPROVED** [review]
LGTM 👍

**Pull request merged**